test/backend/pool.in.cpp

```diff
@@ -1374,7 +1374,7 @@ NGRAPH_TEST(${BACKEND_NAME}, avg_pool_bprop_2d_2channel_2image_dyn_shape)
                                                                padding_below,
                                                                padding_above,
                                                                window_shape,
-                                                               false);
+                                                               true);
```

Comment:
Changed because `Include padding in average not yet implemented in PlaidML` error for `plaidml` backend. In `master` version is bug.
`master`: `exclude_pad = tmp->get_exclude_pad();`
`current:` `include_padding_in_avg_computation = !tmp->get_exclude_pad();`
This param is passed to `AvgPoolBackprop:v0` during downgrade pass